codex/units.nim

```diff
@@ -42,6 +42,8 @@ divMaths(NBytes)
 proc `$`*(ts: NBytes): string = $(int(ts)) & "'NByte"
 proc `'nb`*(n: string): NBytes = parseInt(n).NBytes
 
+logutils.formatIt(LogFormat.textLines, NBytes): $it
+logutils.formatIt(LogFormat.json, NBytes): $it
```

Comment:
Wow, that's a lot shorter 🤣

Reply:
Actually, this can be even shorter: `logutils.formatIt(NBytes): $it`. Will update.

Reply:
Updated.